gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/footer.html

```diff
@@ -1,52 +1,75 @@
 {% load i18n %}
 {% load get_latest_git_hash from simple_filters %}
+{% load get_pages from ndf_tags %}
 
+{% get_pages "Info page" as infopages %}
 <footer>
-  <section class="row">
-    <div class="medium-10 columns ">
-      <div class="row">
-        <div class="medium-12 columns">
-          <ul class="inline-list">
-            <li><a href="{{site.ABOUT}}">{% trans "About" %}</a></li>
-            <li><a href="{{site.PARTNERS}}">{% trans "Partners" %}</a></li>
-            <li><a href="{{site.GROUPS}}">{% trans "Groups" %}</a></li>
-            <li><a href="{{site.TERMS_OF_SERVICE}}">{% trans "Terms of Service" %}</a></li>
-            <li><a href="{{site.PRIVACY_POLICY}}">{% trans "Privacy Policy" %}</a></li>
-            <li><a href="{{site.CONTRIBUTE}}">{% trans "Contribute" %}</a></li>
-            <li><a href="{{site.CONTACT}}">{% trans "Contact" %}</a></li>
+    <div class="myfooter">
+    <div class="footer-section container content-section">
+        <div class="row" style="padding-left:0px;">
+            <div class="footer_content">
+                <img class="footer-logo" src="/static/ndf/css/themes/clix/logo.svg">
```

Comment:
Please use `{{site.logo}}` in order to get logo as per settings,Don't use static logo path.